app/src/lib/api.ts

```diff
@@ -231,6 +231,17 @@ function toGitHubIsoDateString(date: Date) {
   return date.toISOString().replace(/\.\d{3}Z$/, 'Z')
 }
 
+type UserWithTokenScopes = {
```

Comment:
Just curious why you chose a type here instead of an interface?

Reply:
Stylistic reasons mostly, not something I'm hugely attached to.

This is also something internal to Desktop rather than an API response, so I avoided following that pattern here.